src/main/java/com/napier/sem/App.java

```diff
@@ -17,112 +17,39 @@
-        // 6.1 - Produce a report on the top N capital cities in the world organised by largest population to smallest
-        // ArrayList<Record> records = dao.topNCapitalCitiesIn(WORLD, "",5);
-        // 6.2 - Produce a report on the top N capital cities in a continent organised by largest population to smallest
-        // ArrayList<Record> records = dao.topNCapitalCitiesIn(CONTINENT, "Asia", 5);
-        // 6.3 - Produce a report on the top N capital cities in a region organised by largest population to smallest
-        // ArrayList<Record> records = dao.topNCapitalCitiesIn(REGION, "Caribbean", 7);
-
-        // Use-Cases 7.1-7.3
-        // 7.1 - Produce a report on the population of people, people living in cities, and people not living in cities in each continent
-        // ArrayList<Record> records = dao.populationLivingInAndNotInCities(CONTINENT, "Europe");
-        // 7.2 - Produce a report on the population of people, people living in cities, and people not living in cities in each region
-        // ArrayList<Record> records = dao.populationLivingInAndNotInCities(REGION, "Caribbean");
-        // 7.3 - Produce a report on the population of people, people living in cities, and people not living in cities in each country
-        // ArrayList<Record> records = dao.populationLivingInAndNotInCities(COUNTRY, "France");
+        UserPrompt prompt = new UserPrompt(dao);
 
-        // Use-Cases 8.1-8.6
-        // 8.1 - Produce a report on the population of the world
-        // ArrayList<Record> records = dao.populationOf(WORLD, "");
-        // 8.2 - Produce a report on the population of a continent
-        // ArrayList<Record> records = dao.populationOf(CONTINENT, "Europe");
-        // 8.3 - Produce a report on the population of a region
-        // ArrayList<Record> records = dao.populationOf(REGION, "Caribbean");
-        // 8.4 - Produce a report on the population of a country
-        // ArrayList<Record> records = dao.populationOf(COUNTRY, "Spain");
-        // 8.5 - Produce a report on the population of a district
-        // ArrayList<Record> records = dao.populationOf(DISTRICT, "Scotland");
-        // 8.6 - Produce a report on the population of a city
-        // ArrayList<Record> records = dao.populationOf(CITY, "London");
-
-        // Use-case 9.1
-        // Produce a report on the number of who speak the following specific languages from greatest number to smallest, including the percentage of the world population
-        // ArrayList<Record> languages = dao.languageReport();
-
-        //Display results
-        for (Record record : records) System.out.println(record);
+        try
+        {
+            prompt.start();
+        }
+        catch (NoSuchElementException e)
+        {
+            System.out.println(App.HORIZONTAL_LINE + "\nNo console detected, running example query");
+            for (Record record: dao.topNCountriesIn(App.REGION, "Caribbean", 5)) System.out.println(record);
+        }
```

Comment:
this is to prevent travis from hanging because the docker container will run the app and it won't have a console or a user to enter commands.